runtime/compiler/runtime/MetaDataDebug.cpp

```diff
@@ -579,7 +585,13 @@ TR_Debug::printByteCodeStack(int32_t parentStackIndex, uint16_t byteCodeIndex, c
                              J9_CLASS_FROM_METHOD(ramMethod)->romClass,
                              J9_BYTECODE_START_FROM_RAM_METHOD(ramMethod),
                              byteCodeIndex, byteCodeIndex, flags, bcPrintFunc, this, indent);
-      sprintf(indent, "%s   ", indent);
+      static const char indentString2[] = "   ";
+      size_t indentSize2 = sizeof(indentString2) - 1;
+      size_t indentLen = strlen(indent);
+      if ((bufLen > indentSize2) && (indentLen < (bufLen - indentSize2)))
+         {
+         strncpy(indent, indentString2, indentSize2 + 1);
```

Comment:
The destination should be `&indent[indentLen]` (we want to append here).

Reply:
Updated.